java/mykola-melnyk/src/com/zoolatech/lecture5/tasks/_3/Task3.java

```diff
@@ -0,0 +1,29 @@
+package com.zoolatech.lecture5.tasks._3;
+
+import java.util.*;
+
+/**
+ * Create a method that accepts a list of words, and displays only unique one (order can be random). Bonus task:
+ * treat strings in a different case as the same one: “Apple” and “apple” should be treated as the same string
+ * (don’t use any methods that create a new string object - like toUpperCase or toLowerCase - you can work only
+ * with a given input).
+ * Input: [“apple”, “orange”, “pear”, “apple”, “banana”, “orange”]
+ * Output: apple
+ * orange
+ * pear
+ * banana
+ */
+
+public class Task3 {
+
+    public static Set<String> uniqStr7(String... words) {
+        Comparator<String> comparator = new CaseAgnosticComparator()::compare;
```

Comment:
you don't need to use a method reference there - you could assign an instance of `CaseAgnosticComparator` directly

Reply:
Done

Reply:
it seems you haven't pushed your changes